apps/addons/cron.py

```diff
@@ -454,6 +454,7 @@ def reindex_addons(index=None, addon_type=None):
     ids = (Addon.objects.values_list('id', flat=True)
            .filter(_current_version__isnull=False,
                    status__in=amo.VALID_STATUSES,
+                   is_listed=True,
```

Comment:
We may need it in more places. I'll take a look. Probably the `index_addons` could short circuit if the addon is not listed.